qemu/tests/qemu_img.py

```diff
@@ -177,6 +178,42 @@ def create_test(cmd):
                 cluster_size=params.get("image_cluster_size"))
         remove(img)
 
+    def send_signal(timeout=360):
+        """
+        send signal "SIGUSR1" to qemu-img without the option -p
+        to report progress
+        """
+        logging.info("Send signal to qemu-img")
+        end_time = time.time() + timeout
+        while time.time() < end_time:
+            time.sleep(1)
+            status = utils.system("kill -SIGUSR1 `pidof qemu-img`",
+                                  ignore_status=True)
+            if status == 0:
+                return None
+        logging.info("Fail to get pid of qemu-img")
+
+    def check_command_output(CmdResult):
+        """
+        Check standard error or standard output of command
+        : param CmdResult: a list of CmdResult objects
+        """
+        logging.info("Check result of command")
+        check_output = params.get("check_output", "exit_status")
+        output = getattr(CmdResult, check_output)
```

Comment:
I think it' necessary to check check_output, as it is added in configure file by developer when add a new case. If the issue happen, I think it maybe a bug introduced by the case developer.